packages/firestore/src/core/types.ts

```diff
@@ -29,9 +31,7 @@ export type TargetId = number;
 
 export type ListenSequenceNumber = number;
 
-// TODO(b/35918695): In GRPC / node, tokens are Uint8Array. In WebChannel,
-// they're strings. We should probably (de-)serialize to a common internal type.
-export type ProtoByteString = Uint8Array | string;
+export type ProtoByteString = Blob;
```

Comment:
This type declaration doesn't seem to be adding much. If anything it just adds to confusion.

Previously this existed as a separate type to indicate that it was this funky either-or type, but now it seems like it's mostly just a wart. There's no distance between these types and it's confusing to know when to pick which (as your PR shows).

Reply:
Removed from types.ts as we now use ProtoByteString as the default representation